tests/intern.ts

```diff
@@ -29,7 +29,7 @@ export const proxyUrl = 'http://localhost:9001/';
 // Note that the `build` capability will be filled in with the current commit ID from the Travis CI environment
 // automatically
 export const capabilities = {
-	'browserstack.selenium_version': '2.46.0',
+	// 'browserstack.selenium_version': '2.46.0',
```

Comment:
It did while the CI/BrowserStack is being flakey...  It is mainly a reminder that it needs to be addressed before it is landed formally.